sdk/src/storage-settings.ts

```diff
@@ -173,7 +177,8 @@ export class StorageSettings<T extends string> implements Settings {
             }
             return setting.defaultValue;
         };
-        const ret = parseValue(this.device.storage.getItem(key), setting, readDefaultValue, rawDevice);
+        const item = this.device.storage.getItem(key);
```

Comment:
can revert this change to minimize diff, since the original throw you had earlier was removed.

Reply:
Agreed